AsyncDisplayKit/ASMultiplexImageNode.mm

```diff
@@ -325,6 +326,12 @@ - (void)didExitVisibleState
 
 #pragma mark - Core
 
+- (void)setImage:(UIImage *)image
+{
+  ASDisplayNodeAssert(NO, @"Setting the image directly to an ASMultiplexImageNode is not allowed.");
```

Comment:
Should the assert explain why?

"Setting the image directly on an ASMultiplexImageNode is unsafe. It will be cleared in didExitPreloadRange and will have no way to restore in didEnterPreloadRange"